### PR TITLE
build 0.20.rc as rc

### PR DESCRIPTION
**Pull request by amueller**
ok so this is #70 but using the RC tag and merging into the RC branch. fingers crossed?

### CODE REVIEW

**Comment by amueller** [comment]
@jakirkham can you give me a hand? I'm not sure where to set the channel for circle.

File: conda-forge.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
+    BINSTAR_TOKEN: xHG5feKFSl87s9IK6ReTanlNhHkXp4xw5+pdNJxXzpUOHhhDznnJj7+929IxeEuoHJToVty+6BpbLOZGC4B/EirIxY7csjf5ymWHrcTo/hfQ5PL3nlTseoL5l2p2pPwFvzrBREt0O5BBCeiM2rMbr2m814d+mx/VSXvFttfrBbrK3Yn2aU4xdyU0Ce2Rap7QMO9PGwvb1quZWfdyD4DGVx3v6qap+L9IPmOUzJ+Iut58EXD04h3yfdKYulcqv7rlAlQJWokqrlKBsKfRlbP1MaYnKPkx2fcsCneH+8ItJCjoHnTRcGVj6kC6PtQP5cQkScWcWLJPzGhggAywgoTqAGTaGkxm/zEMbObXQGhnEgFcQaD5kn6lEkRFlggobVMDPf8C6wanXqh037So0Qyn/IFqDE5/YyhqPfM2/dgeu7VKEqhqqtK/1/pUiicmDMCKLIbCN9v7aYylZMVWi9BDxubP0hyNz/PssCdu6XmSwwUGgsZVr3JqXWPWX4mskwnny1S45BDzdj1dZaRCeWnzhdN1NeooBZWFH6LW9ojD014rD58TejscKYQkZfOlgbAG5lj4fD8urEBdp8ntLySJywN2gCpbFzvBkyBen6ilDgp5A0H3mOEohPEV53q9XdIXbayeLK524B/eLGmN5s2vpGNh4Hdm0CKg0TYRfwxhWU4=
 appveyor:
   secure:
-    BINSTAR_TOKEN: MP4hZYylDyUWEsrt3u3cod2sbFeRwUziH02mvQOdbjsTO/l1yIxDkP/76rSIjcGC
+    BINSTAR_TOKEN: tumuXLL8PU75WMnRDemRy02ruEq2RpNxeK3dz0MjFssnosPm2v4EFjfNB4PTotA1
 channels:
   targets:
-  - [conda-forge, rc]
+    - [conda-forge, rc]
```
Comment (jakirkham) [inline conda-forge.yml]: Hmm...this should be sufficient. 😕

File: .ci_support/linux_python2.7.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+c_compiler:
+- toolchain_c
+channel_sources:
+- conda-forge,defaults
+channel_targets:
+- conda-forge main
```
Comment (jakirkham) [inline .ci_support/linux_python2.7.yaml]: Yet this doesn't appear correct. Thoughts @conda-forge/core?

File: conda-forge.yml (hunk 1 above)
Comment (amueller) [reply conda-forge.yml]: indentation problem?

Comment (amueller) [reply conda-forge.yml]: removed the spaces, ``conda smithy rerender`` said "no changes". It also said "no changes" when I added the line.

Comment (jakirkham) [reply conda-forge.yml]: For clarity, what you have here seems correct to me. Think `conda-smithy` is ignoring it for some reason, but don't fully understand why. Though there have been some changes with channels in recent `conda-smithy` releases, which could have changed how this gets handled. Let's see if someone else can spot what is going on. 😄

Comment (amueller) [reply conda-forge.yml]: is there a way to force a rebuild?

**Comment by amueller** [comment]
I'm possibly blind but I don't see ``targets`` being used in ``conda-smithy``.

I would have expected it to show up somewhere in ``appveyor.yml.tmpl`` but it doesn't.

**Comment by CJ-Wright** [comment]
@conda-forge/core maybe that works?

**Comment by jakirkham** [comment]
Are any changes needed for macOS?

**Comment by CJ-Wright** [comment]
I think the changes went in right?

**Comment by amueller** [comment]
this is where I check that? https://anaconda.org/conda-forge/scikit-learn/labels

**Comment by jakirkham** [comment]
Sorry was on my phone earlier so couldn't check.

This should work, but it's empty currently.

ref: https://anaconda.org/conda-forge/scikit-learn/files?version=0.20rc1

**Comment by jakirkham** [comment]
Also thanks for doing this @amueller. 😄

**Comment by CJ-Wright** [comment]
Looks like this is working

**Comment by amueller** [comment]
yeah worked :)

**Pull request merged**